### ISSUE TITLE
Allow uploading of App Previews in deliver

### ISSUE THREAD

**Issue by jnutting** [fastlane/fastlane#15515]
### Feature Request
`fastlane deliver` already lets us upload prepared screenshots. It would be great if it also let us upload App Preview videos in the same automated way. As of 2.134.0, `deliver` ignores App Preview files while uploading screenshots.

#### Motivation Behind Feature
More and more app teams are supplementing their screenshots with videos, and have the same logistics problems as with screenshots (dealing with multiple sizes and localizations). Avoiding the drudgery of dragging these into App Store Connect would be great.

#### Feature Description
Basically, the idea is just to make `fastlane deliver` treat app previews just like screenshots stored in the `fastlane/screenshots` subdirectories, so we can push prepared content to ASC with a single command. This has been discussed previously in #9532, #9660, #9775, and some related work has been done in #10546, but this is still an unsolved problem.

#### Alternatives or Workarounds
Alternatives include just using ASC in the traditional way, or scripting a separate delivery step for App Previews by using `spaceship`.

**Comment by sprzedwojski** [comment]
I upvote that so much! 👏 🔥 💪 
I need to implement uploading App Previews via Fastlane in a project, and using `deliver` would be perfect, as this is exactly how we upload screenshots and other metadata, similarly to what @jnutting said.

I also don't know Ruby at all so I can't really help here... :/
